doc/releases/nautilus.rst

```diff
@@ -1,3 +1,104 @@
+v14.2.11 Nautilus
+=================
+
+This is the eleventh release in the Nautilus series. This release brings a
```

Comment:
technically it's the twelfth since we started at 0 :)

Reply:
made it the eleventh backport release :)